gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -648,7 +570,10 @@ gst_tensor_filter_common_get_property (GstTensorFilterPrivate * priv,
       g_value_set_string (value, prop->fwname);
       break;
     case PROP_MODEL:
-      g_value_set_string (value, prop->model_file);
+      if (prop->num_models > 0)
+        g_value_set_string (value, prop->model_files[0]);
```

Comment:
Question: IMO, we need to print all model files when called get_property(). plz update this later.

Reply:
Oh, I see. then, is it fine to return a comma-separated string?

Reply:
I think it is ok, other properties like dimension, type also returns comma-separate string.

Reply:
Got it

Reply:
@jaeyun-jung This feature is updated in the same PR. Please check this.